client/js/components/fragments/Header.js

```diff
@@ -10,24 +10,21 @@ class Header extends Component {
   render() {
     const { user, onLogout, onLogoutSuccess } = this.props;
 
-    var header;
+    let navigation;
     if (user.id) {
-      header = <HeaderAuthenticated user={user} onLogout={onLogout} onLogoutSuccess={onLogoutSuccess}/>;
+      navigation = <HeaderAuthenticated user={user} onLogout={onLogout} />;
```

Comment:
If onLogoutSuccess is not needed plese remove also the props that are not used any more and the callback that is passed here, as it won't be used anymore (App.js:redirectToHomePage)